backend/apps/common/views.py

```
@@ -28,6 +28,94 @@
+
+
+def _apply_resident_password_or_reset(user, data: dict, request) -> None:
+    try:
+        passwd = data.get("password")
+        if passwd:
+            user.set_password(passwd)
+            user.save()
+            return
+        process_password_reset_request(user.email, request)
+    except Exception:
+        pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)